provisioning/roles/mangaki_sync/tasks/main.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Ensure static files directory to be present
+  file:
+    owner: '{{ mangaki_sync_user }}'
+    group: '{{ mangaki_sync_group }}'
+    mode: 'u=rwx,g=rx,o=rx'
+    path: '{{ mangaki_source_static_root }}'
+    state: 'directory'
+  become: true
+  run_once: true
+  when: 'mangaki_sync_collectstatic|bool'
+  tags: action
```

Comment:
I don't think this should be an action, or that it should be in `mangaki_sync`. `mangaki_sync` is intented for destructive changes to the state of the machine (to get it up-to-date); conceptually, the creation of this directory is a declarative way of stating some information about the machine -- if the machine is intended to be used for syncinc, then the directory should always be present, whether we are presently collecting staticfiles or not.